pkg/k8s/apis/cilium.io/client/crds/v1alpha1/cilium.io_tracingpolicies.yaml

```diff
@@ -192,6 +192,10 @@ spec:
                                     action
                                   format: int32
                                   type: integer
+                                argSig:
+                                  description: A signal number for signal action
+                                  format: int32
```

Comment:
Can we make please a string here? so we can handle both "10" or "SIGUSR1" as an example; all of them start with "SIG*"

Reply:
good idea, is that ok as follow up change?

Reply:
I also think suggestion make sense.
Either option (doing the in this PR or as a followup) is fine with me.
Also, found the the function we can use: https://pkg.go.dev/golang.org/x/sys/unix#SignalNum.